src/objecttypes/token/models.py

```diff
@@ -6,7 +6,7 @@
 
 
 class TokenAuth(models.Model):
-    token = models.CharField(_("token"), max_length=40, primary_key=True)
+    token = models.CharField(_("token"), max_length=40, db_index=True)
```

Comment:
```suggestion
    token = models.CharField(_("token"), max_length=40, unique=True)
```

This should be unique to prevent multiple having the same token. If unique, its also indexed.